config/routes.rb

```diff
@@ -9,6 +9,7 @@
   get '/onboarding', to: 'onboarding#index'
   post '/onboarding', to: 'onboarding#create'
   get '/onboarding/success', to: 'onboarding#success'
+  post '/onboarding/invite-url', to: 'onboarding#create_invite_url'
```

Comment:
I think it’s a Rails convention to use underscores instead of dashes. (I personally dislike that, but for the sake of consistency, I’d go with underscores… 😄)

Reply:
rails convention to use underscores in url's? that seems odd... I'd have to look better into that, but I believe underscores would be the anti-pattern. See https://www.woorank.com/en/blog/underscores-in-urls-why-are-they-not-recommended

Reply:
Yeah, I don’t like that either, but I think if you had a multi-word resource (e.g. `OnboardingInvite`) the url helpers and everything would create a path like `/onboarding_invite` if you don’t explicitly pass in the path with dashes.

I’m not 100% sure if that’s still the case though – will try it out.

Reply:
Also I’m not sure if Google is still unable to interpret underscores in URLs in 2020. Even if they weren’t, I’d try to stick with the Rails way as SEO isn’t a priority for our app.

But let me check if what I said about underscores being a Rails convention is true in the first place… 😄

Reply:
Yeah, I would generally stick to the Rails way, but in this case I think the world-wide web is much bigger than Rails and it seems to be an anti-pattern to use underscores in urls, so I would prefer to shorten it to `/onboarding/invite` than to follow aimlessly the Rails way.

Curious what others think, and what you find out. Maybe, I am overstating the underscores in urls being an anti-pattern.

Reply:
> so I would prefer to shorten it to /onboarding/invite than to follow aimlessly the Rails way.

Yes, that sounds like a good solution (maybe even better than `create-invite` in the first place…)

Sounds like a good solution!

> Maybe, I am overstating the underscores in urls being an anti-pattern.

Even though I’m not sure how important it is from a technical and/or UX perspective, I think it just looks ugly! 😄

Reply:
Yeah, I don't think it's important from that perspective. And it's a post endpoint that isn't exposed for SEO, so even from that aspect it's not important, but I agree that it raises my eyebrows, in a similar way that you can use underscores in class names as well, but it's widely considered best practice to use hyphens.